back/engines/commercial/bulk_import_ideas/app/models/bulk_import_ideas/patches/idea.rb

```diff
@@ -40,7 +40,7 @@ def update_idea_import
       end
 
       def remove_import_author
-        return unless idea_import&.user_created == true && publication_status == 'draft'
+        return unless author && idea_import&.user_created == true && publication_status == 'draft'
```

Comment:
No, this method should only be used against users who have just been created through the import. Maybe I'll add a check to see if they have a password or sso set just in case the idea has been sat in draft for a while and indeed a user has then logged in with it.

Reply:
Just for reference, I know of two places where we delete such empty users.

https://github.com/CitizenLabDotCo/citizenlab/blob/3b947dd6bef96f415ba3cf6224e0d79e6b80685a/back/engines/commercial/verification/app/services/verification/verification_service.rb#L107-L109

https://github.com/CitizenLabDotCo/citizenlab/blob/3b947dd6bef96f415ba3cf6224e0d79e6b80685a/back/app/services/authentication_service.rb#L44-L47